scss/components/_case-files.scss

```diff
@@ -8,13 +8,24 @@
       content: none;
     }
   }
+
+  .civicase__bulkactions-message {
+    margin: 0 85px 10px;
+
+    .alert {
+      border-bottom: 1px solid $crm-grayblue-dark !important;
+      box-shadow: none;
+    }
+  }
+}
+
+.civicase__file-tab-filters {
+  display: inline-block;
+  float: right;
 }
 
 .civicase__file-tab-actions {
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-end;
-  margin-bottom: 24px;
+  margin-bottom: 20px;
```

Comment:
Is it possible to use this class instead of having a custom hard coded one?

https://github.com/compucorp/uk.co.compucorp.civicase/blob/9b51bdac11a30e2ae8367b9b323d2e0d9f5404a6/scss/components/_case-tab.scss#L5-L7

In this way we have the same spacing for all tabs.